tasks/main.yml

```
@@ -155,6 +156,7 @@
         basedir: /
         strip: 1
         backup: yes
+      ignore_errors: yes
```

We don't want to ignore errors here; we want to catch failures from this during weekly testing.

If a user runs into an error with this task and they don't want to fix it, they can just set `pve_remove_subscription_warning` to false.

Not to mention this isn't really related to this PR, anyway.

---

oups ... sorry I don't see that, it just because it was in error when I execute it and for me, this is not a big deal if it's wrong ... but get it. I remove it !